### PR TITLE
Lowers ghosts and insain clowns hp to 120 from 1e6

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request

Lowers the npc shade ghost things that spawn in lockers to 120 as well as insain clowns

## Why It's Good For The Game

They cant really be killed and theirs not reason for that, they can float onto cargo shuttle and stop it from going

## Changelog
:cl:
tweak: shades and clowns HP
/:cl:

### CODE REVIEW

**Comment by Ghommie** [comment]
This alone won't stop their unkillable status. 
Their Initialize()'s godmode and `adjustHealth()` ovverride also make them invincible to anything short of admin intervention. I just looked at the file a couple days ago and I'm spooked by the potential sentience potion abuse on top of other things.
Regardless, if you want to make those shades pullable, you should consider removing the `anchored = TRUE`.

File: code/modules/holiday/halloween/halloween.dm [hunk 1]
```diff
@@ -195,16 +194,15 @@
...
+	health = 120
 	emote_see = list("silently stares")
 	unsuitable_atmos_damage = 0
 	var/timer
 
 /mob/living/simple_animal/hostile/retaliate/clown/insane/Initialize()
 	. = ..()
 	timer = rand(5,15)
-	status_flags = (status_flags | GODMODE)
 	return
```
Comment (silicons) [inline code/modules/holiday/halloween/halloween.dm]: remove this return while you're at it

**Review by silicons — CHANGES_REQUESTED** [review]
like ghom said this won't work because adjustHealth is still overridden, change the . = 0 to . = ..() please.

File: code/modules/holiday/halloween/halloween.dm [hunk 2]
```diff
@@ -176,7 +176,7 @@
 		return
 
 /mob/living/simple_animal/shade/howling_ghost/adjustHealth()
-	. = 0
+	. = ..()
```
Comment (Ghommie) [inline code/modules/holiday/halloween/halloween.dm]: You can just remove this then, all it does is call parent, which it kinda the default.

**Review by silicons — APPROVED** [review]

**Pull request merged**